File: Examples/DotNET/CSharp/DiskUsage/GetDiskUsageFromThirdPartyStorage.cs
using System;
using Com.Aspose.Storage.Api;
using Com.Aspose.Storage.Model;

namespace DiskUsage
{
    class GetDiskUsage
    {
        public static void Run()
        {
            // ExStart:1           
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);           
            string storage = "MyStorageName";            
            try
            {
                // Invoke Aspose.Storage Cloud SDK API to get disc usage
               DiscUsageResponse apiResponse = storageApi.GetDiscUsage(storage);
               if (apiResponse != null && apiResponse.Status.Equals("OK"))
               {
                   Console.WriteLine("Total Size (bytes): " + apiResponse.DiscUsage.TotalSize);
                   Console.WriteLine("Used Size (bytes): " + apiResponse.DiscUsage.UsedSize);
                   Console.ReadKey();
               }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}

File: Examples/PHP/DiskUsage/GetDiskUsageFromThirdPartyStorage.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Storage\StorageApi;
use Aspose\Storage\AsposeApp;

class DiskUsage {

	public $storageApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->storageApi = new StorageApi();
	}

	public function getDiskUsage() {
        $result = $this->storageApi->GetDiscUsage($storage = "MyDropboxStorage");        
        print_r($result);
	}
}

$diskUsage = new DiskUsage();
$diskUsage->getDiskUsage();

?>
//ExEnd:
